CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Questions and Help](#questions-and-help)
+1. [Getting Started](#getting-started)
+1. [Useful Resources](#useful-resources)
+
+# About the Project
```

Comment:
Some of this is redundant to the main project docs, readme, etc. Might as well not repeat ourselves.

Reply:
Ok, I'll remove some stuff. I took the headings directly from someone's CONTRIBUTING.md template because I wasn't sure what people were expecting to find here.